<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogExportPrimitive</class>
 <widget class="QWidget" name="DialogExportPrimitive">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>372</width>
    <height>325</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>344</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Primitives Export</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="5" column="0">
    <widget class="QLabel" name="label_folder">
     <property name="text">
      <string>Folder</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QRadioButton" name="radioButton_mergeOneFile">
     <property name="text">
      <string>Merge in one file</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="8" column="0" colspan="4">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="pushButton_export">
       <property name="text">
        <string>Export</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_cancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="5" column="1" colspan="2">
    <widget class="QLineEdit" name="lineEdit_folder">
     <property name="placeholderText">
      <string>Enter output folder</string>
     </property>
    </widget>
   </item>
   <item row="11" column="0" colspan="3">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="0" colspan="4">
    <widget class="QCheckBox" name="openExplorerFolderCheckBox">
     <property name="text">
      <string>Open folder after export</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_fileName">
     <property name="text">
      <string>File Name</string>
     </property>
    </widget>
   </item>
   <item row="5" column="3">
    <widget class="QToolButton" name="toolButton_folder">
     <property name="text">
      <string>Select Out Folder</string>
     </property>
     <property name="icon">
      <iconset resource="../../../openscantools.qrc">
       <normaloff>:/icons/100x100/open_project.png</normaloff>:/icons/100x100/open_project.png</iconset>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="2">
    <widget class="QLineEdit" name="lineEdit_fileName">
     <property name="placeholderText">
      <string>Enter output file or header name</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="4">
    <widget class="QGroupBox" name="objectSelection">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Object selection</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="horizontalSpacing">
       <number>20</number>
      </property>
      <item row="0" column="1">
       <widget class="QCheckBox" name="tagsBox">
        <property name="text">
         <string>Tags</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QCheckBox" name="pointsBox">
        <property name="text">
         <string>Points</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Filter</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1" colspan="2">
       <widget class="QComboBox" name="comboBox_filter">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QCheckBox" name="scanBox">
        <property name="text">
         <string>Scans</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QCheckBox" name="viewpointBox">
        <property name="text">
         <string>Viewpoints</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="measuresBox">
        <property name="text">
         <string>Measures</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QCheckBox" name="boxesBox">
        <property name="text">
         <string>Boxes</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QCheckBox" name="pipingBox">
        <property name="text">
         <string>Piping</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="spheresBox">
        <property name="text">
         <string>Spheres</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QCheckBox" name="externBox">
        <property name="text">
         <string>Extern Model</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QCheckBox" name="pcoBox">
        <property name="text">
         <string>PCO</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QToolButton" name="toolButton_file">
     <property name="text">
      <string>Select Out File</string>
     </property>
     <property name="icon">
      <iconset resource="../../../openscantools.qrc">
       <normaloff>:/icons/100x100/save.png</normaloff>:/icons/100x100/save.png</iconset>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QRadioButton" name="radioButton_oneFilePerObjectType">
     <property name="text">
      <string>One file per object type</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="4">
    <widget class="QCheckBox" name="restoreTruncatedCoordinatesCheckBox">
     <property name="text">
      <string>Truncated coordinates: restore original coordinates</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>scanBox</tabstop>
  <tabstop>tagsBox</tabstop>
  <tabstop>pointsBox</tabstop>
  <tabstop>measuresBox</tabstop>
  <tabstop>boxesBox</tabstop>
  <tabstop>pipingBox</tabstop>
  <tabstop>spheresBox</tabstop>
  <tabstop>externBox</tabstop>
  <tabstop>pcoBox</tabstop>
  <tabstop>viewpointBox</tabstop>
  <tabstop>comboBox_filter</tabstop>
  <tabstop>radioButton_mergeOneFile</tabstop>
  <tabstop>radioButton_oneFilePerObjectType</tabstop>
  <tabstop>lineEdit_fileName</tabstop>
  <tabstop>toolButton_file</tabstop>
  <tabstop>lineEdit_folder</tabstop>
  <tabstop>toolButton_folder</tabstop>
  <tabstop>openExplorerFolderCheckBox</tabstop>
  <tabstop>pushButton_export</tabstop>
  <tabstop>pushButton_cancel</tabstop>
 </tabstops>
 <resources>
  <include location="../../../openscantools.qrc"/>
 </resources>
 <connections/>
</ui>
